salt/states/pkgrepo.py

```diff
@@ -430,7 +430,7 @@ def managed(name, ppa=None, **kwargs):
                         break
                 else:
                     break
-            elif kwarg == 'comps':
+            elif kwarg in ['comps', 'key_url']:
```

Comment:
Mind changing this to a tuple?
```suggestion
            elif kwarg in ('comps', 'key_url'):
```